example/fruits/viewsets.py

```diff
@@ -0,0 +1,16 @@
+from django.shortcuts import render
+
+from rest_framework import viewsets
+
+from rest_framework_simple_api_key.backends import APIKeyAuthentication
+from rest_framework_simple_api_key.permissions import IsActiveEntity
+from rest_framework.response import Response
+
+
+class FruitViewSets(viewsets.ViewSet):
+    http_method_names = ["get"]
+    authentication_classes = (APIKeyAuthentication,)
```

Comment:
Add a comment to show that, it's here you import and use the package and remove the trailing comma, because didn't see the role

Reply:
As I am using tuples, you need to add a traling comma at the end to avoid weird behaviors https://note.nkmk.me/en/python-tuple-single-empty/#tuple-with-one-element